runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -265,7 +265,11 @@ MM_ScavengerDelegate::getObjectScanner(MM_EnvironmentStandard *env, omrobjectptr
 #endif /* defined(OMR_GC_MODRON_STRICT) */
 
 	GC_ObjectScanner *objectScanner = NULL;
-	switch(_extensions->objectModel.getScanType(objectPtr)) {
+	J9Class *clazzPtr = J9GC_J9OBJECT_CLAZZ(objectPtr);
+
+	switch(_extensions->objectModel.getScanType(clazzPtr)) {
+	case GC_ObjectModel::SCAN_MIXED_OBJECT_LINKED:
+		_extensions->scavenger->deepScan(env, objectPtr, clazzPtr->gcPriorityScanField1, clazzPtr->gcPriorityScanField2);
```

Comment:
add a comment after the deepScan(), that we intentionally fall through and scan this as any other mixed object